keps/sig-windows/2258-kubectl-service-log-viewer/kep.yaml

```diff
@@ -0,0 +1,40 @@
+title: KEP Template
+kep-number: 2258
+authors:
+  - "@aravindhp"
+  - "@LorbusChris"
+owning-sig: sig-windows
+participating-sigs:
+  - sig-node
```

Comment:
Given the kubectl enhancements, I guess sig-cli needs to be involved as well and needs to approve?

Reply:
We probably want someone form sig-auth to approve here as well.
At least from the windows side exposing the ability to query arbitrary event logs can expose a lot more system information than what is exposed in kubelet logs.

Reply:
Added `sig-auth` to participating sigs.